coordinator/src/node.rs

```diff
@@ -206,14 +203,22 @@ impl Node {
         let order_id = trade_params.filled_with.order_id.to_string();
         tracing::info!(trader_id, order_id, "Executing match");
 
+        //TODO(holzeis): The coordinator leverage should come from the selected liquidity option.
+        let coordinator_leverage = 2.0;
```

Comment:
:thinking: Surely it is leverage 1?! Or have we decided to change this already? We should probably change it in a dedicated patch if so.

Reply:
it was leverage 1.0 but with the new liquidity options its always 2.0.